### PR TITLE
Nouveau membre

### PR DESCRIPTION
**Pull request by Delphineray**
Je souhaite rejoindre la communauté des baggers

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -17686,7 +17686,60 @@ var data = {
...
+    {
+      since: '2020-09-04',
+      name: 'Delphine Raymond',
+      bio:
+        "Product Owner & Product Manager depuis plus de 7 ans, je travaille chez Lucca et j'ai créé le meetup Product People Nantes, devenu LPCx Nantes, une communauté de PO/PM pour les amoureux de la création de produits numériques.",
+      picture:
+        'https://secure.gravatar.com/avatar/ddbeaab3b87a434071ddc6bfb5af89a8?size=256',
+      websites: [
+        {
+          name: '@Delphine_Ray',
```
Comment (dadoonet) [inline baggers.js]: It should be Twitter here

File: baggers.js [hunk 2]
```diff
@@ -17686,7 +17686,60 @@ var data = {
...
+          url: 'https://twitter.com/Delphine_Ray',
+        }, 
+        {
+          name: 'LinkedIn',
+          url: 'https://www.linkedin.com/in/delphineraymond/',
+        },       
+        {
+          name: 'Lucca',
```
Comment (dadoonet) [inline baggers.js]: It should be Web. See https://github.com/brownbaglunch/bblfr_data#websites

File: baggers.js [hunk 3]
```diff
@@ -17686,7 +17686,60 @@ var data = {
...
+        },
+      ],
+      cities: ['Nantes'],
+      contacts: {
+        twitter: 'Delphine_Ray',
+        mail: 'delphine.raymond@gmail.com',
+      },
+    },    
   ],
+
```
Comment (dadoonet) [inline baggers.js]: Could you remove this blank line?

**Review by dadoonet — CHANGES_REQUESTED** [review]
Left some comments. 
Could you please fix that and I'll be happy to merge it. 😊

File: baggers.js (hunk 1 above)
Comment (Delphineray) [reply baggers.js]: Done

File: baggers.js (hunk 2 above)
Comment (Delphineray) [reply baggers.js]: Done

File: baggers.js (hunk 3 above)
Comment (Delphineray) [reply baggers.js]: Done

**Comment by Delphineray** [comment]
@dadoonet Thanks for your review, I fixed your comments :)

**Comment by Delphineray** [comment]
> Left some comments.
> Could you please fix that and I'll be happy to merge it. 😊

Hello @dadoonet, I fixed your comments, could you merge it? Thanks :)

**Comment by dadoonet** [comment]
Welcome!

**Comment by Delphineray** [comment]
Thanks @dadoonet ;)

**Pull request merged**